Description: Fix endianness verification for powerpc
Author: "Fernando Seiti Furusato" <ferseiti@br.ibm.com>
Origin: other, https://bugs.debian.org/750686
Bug-Debian: https://bugs.debian.org/750686
Last-Update: 2014-06-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/include/lcms2.h
+++ b/include/lcms2.h
@@ -192,11 +192,9 @@
 
 #if defined(__powerpc__) || defined(__ppc__) || defined(TARGET_CPU_PPC)
 #   define CMS_USE_BIG_ENDIAN   1
-#   if defined (__GNUC__) && defined(__BYTE_ORDER) && defined(__LITTLE_ENDIAN)
-#       if __BYTE_ORDER  == __LITTLE_ENDIAN
+#   if defined (__GNUC__) && defined(__LITTLE_ENDIAN__)
 //               // Don't use big endian for PowerPC little endian mode
 #                undef CMS_USE_BIG_ENDIAN
-#       endif
 #   endif
 #endif
 
